Project.toml

```diff
@@ -18,12 +18,12 @@ RigidBodyDynamics = "366cf18f-59d5-5db9-a4de-86a9f6786172"
 [compat]
 ColorTypes = "0.7, 0.8, 0.9, 0.10"
 CoordinateTransformations = "0.5, 0.6"
-GeometryTypes = "0.6, 0.7, 0.8"
+GeometryBasics = "0.2, 0.3"
 InteractBase = "0.8, 0.9, 0.10"
 Interpolations = "0.9, 0.10, 0.11, 0.12"
 LoopThrottle = "0.1"
-MechanismGeometries = "0.4, 0.5"
-MeshCat = "0.10, 0.11"
+MechanismGeometries = "0.4, 0.5, 0.6"
```

Comment:
Likewise this can now be `MechanismGeometries = "0.7"` as soon as this is merged: https://github.com/JuliaRegistries/General/pull/29093

(I ended up using 0.6 for the upgrade to Rotations.jl 1.0, so MechanismGeometries v0.7 is the version with GeometryBasics.jl support)